projects/observability/src/shared/components/cartesian/cartesian-chart.component.ts

```diff
@@ -87,13 +99,16 @@ export class CartesianChartComponent<TData> implements OnChanges, OnDestroy {
     }
 
     this.chart = this.chartBuilderService
-      .build<TData>(this.strategy, this.container.nativeElement, this.renderer)
+      .build<TData>(this.strategy, this.container.nativeElement, this.renderer, this.sync, this.groupId)
       .withSeries(...this.series)
       .withTooltip(
         this.chartTooltipBuilderService.constructTooltip<TData, Series<TData>>(data =>
           this.convertToDefaultTooltipRenderData(data)
         )
-      );
+      )
+      .withEventListener(ChartEvent.Hover, data => {
+        this.mousePositionChange.emit(data);
```

Comment:
Who listens to this? We'd still need some service or similar to receive the event and notify the other charts that are supposed to match

Reply:
working on the service part